src/index.html

```diff
@@ -16,17 +16,34 @@
       href="style.css"
     />
   </head>
+
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      class="search__form"
+      data-qa="big"
+    >
+      <input
+        id="big-search-input"
+        class="search__input"
+        type="text"
+        placeholder="Try “Los Angeles“"
+        data-qa="keypress"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      class="search__form"
+      data-qa="small"
+    >
+      <input
+        id="small-search-input"
+        class="search__input"
+        type="text"
+        placeholder="Try “Los Angeles“"
+        data-qa="hover"
+      />
+    </form>
```

Comment:
The `form` elements are missing the `<label>` tags. Labels are important for accessibility as they provide a text description to form controls.